browser_patches/prepare_checkout.sh

```diff
@@ -91,15 +91,13 @@ else
   git remote add $REMOTE_BROWSER_UPSTREAM $REMOTE_URL
 fi
 
-# Check if we have the $BASE_REVISION commit in GIT
+# If not, fetch from REMOTE_BROWSER_UPSTREAM and check one more time.
+git fetch $REMOTE_BROWSER_UPSTREAM $BASE_BRANCH
```

Comment:
Fetch was un-conditional initially, but was causing some tension. Let's try one more time and see if there are complaints.

Reply:
Yeah, let's try that. If it proves annoyingly slow we can always check that browser_upstream/master != BASE_REVISION before fetching.